mqtt/mqtt-bridge/src/lib.rs

```diff
@@ -10,9 +10,11 @@
     clippy::must_use_candidate,
     clippy::missing_errors_doc
 )]
+#![recursion_limit = "256"]
```

Comment:
this is because of loop with select! in pump. I added the loop because the future might be completed when there is no connectivity receiver in the pump

Reply:
can we switch it to the `future::select` maybe?